build.xml

```diff
@@ -1,4 +1,22 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!-- 
+  NOTICE: Copyright (c) 2017-2023 by the Music Encoding Initiative (MEI) Board (formerly known as "MEI Council").
```

Comment:
Is this `build.xml` this old? I think it showed up at a later point.

Reply:
You are right, @rettinghaus . It was created in late 2020, cf. https://github.com/music-encoding/music-encoding/pull/740/commits/b7a3e0ae1ee7ca86a4c93e3f8362775fa01fc178

Reply:
This is a blanket copyright statement for all the files in the project; many of the guidelines files didn't exist in 2017 either. This statement is the same one as the one in `LICENSE`.

Reply:
Having the same statement in all files is fine. Maintaining the exact date of creation in every file could turnout a maintenance nightmare.

Reply:
I'm going to suggest we don't have file-specific license statements, for the reasons @bwbohl mentions. So I will resolve this.